### ISSUE TITLE
Fail to load resource from 'https://acme-v02.api.letsencrypt.org/acme/new-order': urn:ietf:params:acme:error:badNonce

### ISSUE THREAD

**Issue by kimbirkelund** [ffMathy/FluffySpoon.AspNet.EncryptWeMust#33]
I've started getting the following error when renewing certificates:

```
Exception setting up Let's Encrypt certificate: Fail to load resource from 'https://acme-v02.api.letsencrypt.org/acme/new-order'.
urn:ietf:params:acme:error:badNonce: JWS has an invalid anti-replay nonce: "0001nQHjcj8V0RNRp1k9MRB5NheGz8TT0wLMJgZ9gaSqEiQ"
```

It happens in `FluffySpoon.AspNet.LetsEncrypt.Certes.LetsEncryptClient.PlaceOrder`.

Any idea how to solve this?

**Comment by ffMathy** [comment]
That's new. Never seen that before. Let's see if anyone else is having the same issues.

**Comment by kimbirkelund** [comment]
Looking a bit deeper I'm thinking this might actually be an issue with Certes instead. Still investigating.

**Comment by kimbirkelund** [comment]
Never mind, turns out to not be an issue - my problem is elsewhere.

**Comment by eliezer15** [comment]
@kimbirkelund one of our applications is running into a similar issue, do you have any insights of what the problem was? :) 

Thanks in advance!

**Comment by kimbirkelund** [comment]
No, I never did get it working. I switched to using  Azures native certificate creation.

**Comment by MarkCiliaVincenti** [comment]
I'm having this issue.

**Comment by ffMathy** [comment]
I am so sorry, but I have not got a lot of time lately to fix this. PRs are very welcome!

**Comment by MarkCiliaVincenti** [comment]
To be honest I restarted the application and the problem got resolved on
its own but it was giving that error for days before so something got
stuck.

**Comment by ffMathy** [comment]
Interesting. Any idea on the cause? And thank you for sharing this with us.

**Comment by MarkCiliaVincenti** [comment]
Earlier I had errors that it couldn't connect to the resource; I think
let's encrypt was down.

**Comment by MarkCiliaVincenti** [comment]
I was getting this again after a while. Reverted back to using FluffySpoon.AspNet.LetsEncrypt

**Comment by ffMathy** [comment]
The project is not dead. PRs are very welcome, and will be reviewed and merged.

Also, log files etc would be helpful.
